composer.json

```diff
@@ -108,6 +108,7 @@
     "drupal/module_filter": "^4",
     "drupal/module_permissions": "^3",
     "drupal/openid_connect": "3.x-dev",
```

Comment:
@hosomitm2nd We can remove this since now we have https://git.drupalcode.org/project/openid_connect_daccount/-/blob/1.x/composer.json#L14

```suggestion
```

Reply:
@naoi Thank you for your review.
I removed openid_connect.